src/containers/Main.tsx

```diff
@@ -97,9 +97,9 @@ const Main = () => {
 }
 
 const AuthContentWrapper = styled.div`
-  max-width: 500px;
+  // max-width: 500px;
   margin: 0 auto;
-  padding: 60px 20px 120px;
+  // padding: 60px 20px 120px;
```

Comment:
Commented these lines out for now - we'll figure out a permanent solution for this